mypy/server/update.py

```diff
@@ -174,8 +174,8 @@ def __init__(self,
         # for the cache. This is kind of a hack and it might be better to have
         # this directly reflected in load_graph's interface.
         self.options.cache_dir = os.devnull
+        self.options.use_fine_grained_cache = False
```

Comment:
Modifying the options object it's kind of ugly. Maybe at least mention that it gets mutated in the docstring?